i18n/de/docusaurus-plugin-content-docs/current/help-center/using-famedly/messages.mdx

```diff
@@ -309,6 +309,62 @@ Die Nachricht wird dem Administrator Ihrer Organisation gemeldet.
   </TabItem>
 </Tabs>
 
+## Zu Notizen hinzufügen
+
+Sie können Nachrichten und Medien aus anderen Unterhaltungen speichern, indem Sie sie zu <b>Notizen</b> hinzufügen.
+
+<Tabs>
+  <TabItem value="desktop" label="Desktop" default>
+  <ol>
+    <li>Klicken Sie auf die Nachricht, die Sie speichern möchten.</li>
+    <li>Wählen Sie nun <b>Zu Notizen hinzufügen</b> aus der Liste aus.</li>
+    <li>Die Nachricht wird in einem privaten Chat namens <b>Notizen</b> gespeichert.</li>
+    <li>Sie können auf die Schaltfläche <b>Notizen öffnen</b> klicken, die am unteren Rand des Fensters erscheint (und nach einigen Sekunden verschwindet), um nachzusehen.</li>
+  </ol>
+    </TabItem>
+    <TabItem value="mobile" label="Mobile">
```

Comment:
Totally forgot, could you translate the TabItem, too please? Like this:
`<TabItem value="mobil" label="Mobil">`